src/nft/components/collection/MarketplaceSelect.tsx

```diff
@@ -48,6 +50,7 @@ const MarketplaceItem = ({
       removeMarket(value)
       setCheckboxSelected(false)
     }
+    sendAnalyticsEvent(EventName.NFT_FILTER_SELECTED, { filter_type: 'Marketplace' })
```

Comment:
can we define a FILTER_TYPES enum for constants like 'Marketplace', 'Price Range' etc?